content/v2.0/reference/cli/influx/stacks/update.md

```diff
@@ -30,6 +32,16 @@ influx stacks update [flags]
 | `-u` | `--template-url`      | Template URLs to associate with a stack                    | list of strings |                       |
 | `-t` | `--token`             | Authentication token                                       | string          | `INFLUX_TOKEN`        |
 
+{{% warn %}}
+#### Export an updated template
+To prevent accidental changes, we **strongly recommend** exporting a new template
+any time you add additional resources to a stack using the `--addResource` flag
+with the `influx stack update` command.
+The updated stack will differ from the previous template.
```

Comment:
Does this mean the updated stack includes a template with the newly added resources?